### ISSUE TITLE
MongoDB option missing on connection generator

### ISSUE THREAD

**Issue by rybaczewa** [feathersjs/generator-feathers#499]
### Steps to reproduce
```
$ feathers -V
4.2.3

$ feathers g app
(...)

$ feathers g connection
? Which database are you connecting to? (Use arrow keys)
❯ MySQL (MariaDB) 
  PostgreSQL 
  SQLite 
  SQL Server
```
After generating new app, when trying to create connection, mongodb option is missing, despite latest release supposedly fixing it:
https://github.com/feathersjs/cli/pull/197
https://github.com/feathersjs/generator-feathers/pull/498

### System configuration
```
$ node -v
v10.17.0
```

Not sure left out on purpose or if it's only missing in options here:
https://github.com/feathersjs/generator-feathers/blob/7cb9e14a3d1753e791c46ff886e3a6aafa119c5b/generators/connection/index.js#L182..L187

**Comment by daffl** [comment]
Hi @rybaczewa. It's probably just missing from the list but nobody noticed it so far because it will be run automatically if you create a MongoDB service via `feathers generate service`.

**Comment by ntoskrnl-exe** [comment]
When will this be fixed?

I do not want a work around, I would like to know when this will be fixed. Sorry for being blunt and terse, I am sick of running into problems, which are not associated with code I have wrote.

**Comment by daffl** [comment]
This isn't a workaround. You don't need to generate a connection separately, it will do it for you automatically when you generate a service and pick the type.

But let me also be terse and blunt: If you need it fixed anyway, fix it. __All open source code__ you are using is _your responsibility_ (it even says so in the license). The code in question is at https://github.com/feathersjs/generator-feathers/blob/master/generators/connection/index.js the error probably happens when assembling the prompts at https://github.com/feathersjs/generator-feathers/blob/master/generators/connection/index.js#L177. A pull request will be very welcome. Usually they don't come from people that are terse and blunt in GitHub issues but I'd love to be proven wrong.

**Comment by MauriceArikoglu** [comment]
@daffl sorry you guys had to experience this behaviour. Thanks for the great project. I fixed this in the #580 PR
